tests/ui/imports/glob-conflict-cross-crate-3.rs

```diff
@@ -0,0 +1,19 @@
+// check-pass
+// aux-build:glob-conflict-cross-crate-2-extern.rs
+
+extern crate glob_conflict_cross_crate_2_extern;
+
+mod a {
+    pub type C = i32;
+}
+
+use glob_conflict_cross_crate_2_extern::*;
+use a::*;
+
+fn main() {
+    let _a: C = 1;
```

Comment:
Breaking change: code that previously success to compile will now compile successfully, but with a warning.